crates/bevy_pbr/src/render/mesh/mod.rs

```diff
@@ -1146,22 +1117,27 @@ impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMeshBindGroup<I> {
     fn render<'w>(
         _item: &P,
         _view: (),
-        (mesh_index, skinned_mesh_joints): ROQueryItem<'_, Self::ItemWorldQuery>,
-        mesh_bind_group: SystemParamItem<'w, '_, Self::Param>,
+        (mesh_index, skin_index): ROQueryItem<'_, Self::ItemWorldQuery>,
+        bind_groups: SystemParamItem<'w, '_, Self::Param>,
         pass: &mut TrackedRenderPass<'w>,
     ) -> RenderCommandResult {
-        if let Some(joints) = skinned_mesh_joints {
-            pass.set_bind_group(
-                I,
-                mesh_bind_group.into_inner().skinned.as_ref().unwrap(),
-                &[mesh_index.index(), joints.index],
-            );
+        let error_message = "The MeshBindGroups resource wasn't set in the render phase. \
```

Comment:
why not just inline it?

Reply:
Inlining it would make the code much less compact, Check this out <https://github.com/bevyengine/bevy/pull/8581#discussion_r1207729033>

Reply:
I guess inlining is not necessary, but what I was thinking is to just move it closer to where it is logged in the else branch. It just seems weird where it is now.

To be clear, that's a very minor nitpick.